pyproject.toml

```diff
@@ -20,6 +21,7 @@ target-version = ["py38"]
 [tool.isort]
 line_length = 99
```

Comment:
This might be an internal team convention. But is there any reason we use `99` instead of `79`?